src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -622,6 +622,15 @@ public String getAuthorTitleYear(int maxCharacters) {
     }
 
     /**
+     * Returns the title of the given BibTeX entry as an optional.
+     *
+     * @return the title of a BibTeX entry or null if no title was found.
+     */
+    public Optional<String> getTitle() {
+        return getField(FieldName.TITLE);
```

Comment:
👍 for `Optional<>` - but the javadoc should be updated accordingly: `@return the title of a BibTeX entry or null if no title was found.`

I think this is introduced by copy&paste but we should not introduce new issues with new code. 
Another, rather nitpicky aspect: In another snippet you created variables like `bes` - don't cheat by looking - but what exactly is stored in this variable? :wink: 

Using the name "selectedBibEntries" would be way more self-explanatory...